doc/guides/releases.md

```diff
@@ -326,7 +326,13 @@ grep REPLACEME doc/api/*.md
 and substitute this node version with
 
 ```console
-sed -i "s/REPLACEME/$VERSION/g" doc/api/*.md` or `perl -pi -e "s/REPLACEME/$VERSION/g" doc/api/*.md
+sed -i "s/REPLACEME/$VERSION/g" doc/api/*.md
```

Comment:
Maybe it should be noted that the `-i` flag for `sed` is a GNU-specific one, so won't work everywhere (for example it doesn't work on macOS). Maybe that's outside the scope of this PR though.

Reply:
+1, but let's do that in a separate PR.